<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="folderName_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>61</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Name of the folder:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="folderDestination_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>89</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Folder destination:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="deletionProperties_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>330</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Deletion properties&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="deleteInSubfolders_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete in subfolders</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="askBeforeDeletion_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ask before each deletion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>205</y>
      <width>330</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="deleteHiddenFolders_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>175</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete hidden folders</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>62</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="folderName_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="select_pushButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="folderDestination_lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="browse_pushButton">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
